src/bp-media/bp-media-filters.php

```diff
@@ -2677,3 +2679,29 @@ function bb_setup_template_for_media_preview( $template ) {
 
 	return trailingslashit( buddypress()->plugin_dir ) . 'bp-templates/bp-nouveau/includes/media/preview.php';
 }
+
+/**
+ * Function will check content empty or not for the media.
+ * If content will empty then return true and allow empty content in DB for the media.
+ *
+ * @param bool         $validated_content Boolean from filter.
+ * @param string       $content           Message content.
+ * @param array|object $id                Message ID.
+ *
+ * @return bool
+ */
+function bb_check_is_get_message_content_validate_with_media( $validated_content, $content = '', $message_id ) {
+	if ( ! empty( $content ) ) {
```

Comment:
@milanhirapra Why we need to check empty content again it's already checked before

Reply:
@dipskakadiya I removed this condition from the add filters.